packages/core/src/Core.ts

```diff
@@ -49,7 +49,7 @@ export class Core<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         if (!range.isCollapsed()) {
             range.empty();
         }
-        if (range.startContainer.breakable) {
+        if (this.editor.isBreakable(range.startContainer)) {
```

Comment:
`this.mode.isBreakable` :D

Reply:
Uuuuh... no :shrug:

Reply:
Actually it make more sense. You only need the mode, not the editor. the `editor.isBreakable` is calling the `mode.isBreakable`...

Reply:
No, `editor.isBreakable` calls `editor.plugins.get(Mode).isBreakable`.
In any case I already mentioned that I was open to having a default mode and therefore handling these through the mode every time.

Reply:
This echoes to one of my first comments about either supporting mode at the editor level or accessing it from the plugin rather than "hiding" it thought a dedicated api on the editor.